app/workers/scheduler/for_you_mammoth_scheduler.rb

```diff
@@ -16,6 +16,9 @@ class Scheduler::ForYouMammothScheduler
   Rails.logger.warn "ForYouMammothScheduler LOAD TEST:: x#{LOAD_TEST_MULTIPLIER}" if LOAD_TEST_MULTIPLIER > 1
 
   def perform
+    # Check for existing UpdateForYou Workers first
+    return if ForYouMammothServiceCheck.new.call
```

Comment:
Can we add some logging here to note this situation?
Maybe even an AppSignal? (not sure if overkill)

Reply:
💥 Captured in both sidekiq ui & AppSignal with Trigger
![Firefox Developer Edition 2023-11-20 at 16 48 17@2x](https://github.com/TheBLVD/moth.social/assets/76360/368361d2-7fa0-4b9a-ad1d-bf71d8b9a2f9)
![Safari 2023-11-20 at 16 50 19@2x](https://github.com/TheBLVD/moth.social/assets/76360/0fb0e8a3-7822-475e-bc42-e19aef682dd7)